pkg/controllermanager/apis/config/types.go

```diff
@@ -100,6 +102,14 @@ type BastionControllerConfiguration struct {
 	MaxLifetime *metav1.Duration
 }
 
+// CertificateSigningRequestControllerConfiguration defines the configuration of the CertificateSigningRequest
+// controller.
+type CertificateSigningRequestControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events.
```

Comment:
Copy+paste mistake 😄

Reply:
For all the controllers I can see the same description for `ConcurrentSyncs` field. What should we mention instead here ?

Reply:
I think @timebertt thought you copied from the `EventControllerConfiguration`. However, in this case, it should be fine